mad-dog/src/index.js

```diff
@@ -167,7 +169,8 @@ async function main () {
           numUsers: 1
         }
       )
-      const tests = [signUpReplicaSetTest]//, test, ...blacklistTests]
+
+      const tests = [test, ...blacklistTests, signUpReplicaSetTest]
```

Comment:
we should add this to be run as part of the `npm run start test` cmd as well, not just in `test:ci` - also ideally this would not be a new test `signUpReplicaSetTest` and instead be packaged into the core `consistency_1` test. blacklist tests being separate make sense but this is core functionality and should be treated as such. anyway, we don't have time to fix this now - so lets just make a comment to consolidate after

Reply:
lets punt this to refactor mad dog task for next sprint

Reply:
good point!

Reply:
todo: consolidate test scenarios + add replica set test into core system test file for next sprint. leave alone for this sprint